stable/mlrun/values.yaml

```diff
@@ -69,6 +69,24 @@ api:
   # runAsNonRoot: true
   # runAsUser: 1000
 
+  readinessProbe:
+    httpGet:
+      path: /api/healthz
+      port: http
+    initialDelaySeconds: 10
+    timeoutSeconds: 30
+    periodSeconds: 5
+    failureThreshold: 10
+
+  livenessProbe:
+    exec:
+      command:
+        - /bin/bash
+        - -c
+        - pgrep -f mlrun
+    initialDelaySeconds: 60
+    timeoutSeconds: 10
```

Comment:
missing `periodSeconds`

Reply:
Not missing, I want the default - 10

Reply:
Since you're stating the rest of the values, please state this too explicitly (1-clarity, 2-to keep this unchanged if k8s change default)